utils/loopPages.js

```diff
@@ -0,0 +1,16 @@
+const { Observable } = require('rx');
+
+const readDir = require('./readDir');
+
+module.exports = function loopPages(handler, dir = 'src/pages') {
+  return Observable.from(readDir(dir)).flatMap(articleParentDir => {
```

Comment:
`SuperNit`

`.flatMap()` on a new line please and indent its function correctly 👍

Reply:
@Bouncey 🙌 I like nits, they mean you care about the quality of the code. Updated!